packages/elements/src/overlay/__test__/managers/focus-manager.test.js

```diff
@@ -73,7 +73,7 @@ describe('overlay/manager/FocusManager', () => {
           expect(size()).to.equal(0, 'element should be deregistered');
         });
 
-        xit('Test with activeElement', async () => {
+        it('Test with activeElement', async () => {
```

Comment:
xit is skip all browsers, is it works for modern browsers?

Reply:
Yes, it works well on modern browsers